amplify/backend/function/fetchDomainBalance/src/index.js

```diff
@@ -32,12 +32,32 @@ exports.handler = async (event) => {
       timeframeType,
       timeframePeriodEndDate,
     } = event.arguments?.input || {};
+
+    /**
+     * We want to early return if there is no positive period for which to compute values
+     */
+    if (timeframePeriod <= 0) {
+      return {
+        totalIn: 0,
+        totalOut: 0,
+        total: 0,
+        timeframe: [],
+      };
+    }
+
     const periodForTimeframe = getPeriodFor(
       timeframePeriod,
       timeframeType,
       timeframePeriodEndDate,
     );
 
+    console.log(
```

Comment:
Intentional or a leftover :eyes: ?

Reply:
Intentional, so we can get some more information if there is an error in a specific call